aggregator/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJob.java

```diff
@@ -59,11 +65,13 @@ public void processDailyEmail(Instant scheduledFireTime) {
         }
     }
 
-    List<AggregationCommand> processAggregateEmails(Instant scheduledFireTime) {
-        Instant yesterdayScheduledFireTime = scheduledFireTime.minus(DAILY.getDuration());
+    List<AggregationCommand> processAggregateEmails() {
+        Instant scheduledFireTime = emailAggregationResources.getLastCronJobRun().getLastRun();
+        Instant yesterdayScheduledFireTime = scheduledFireTime.minus(EmailSubscriptionType.DAILY.getDuration());
```

Comment:
Shouldn't `scheduledFireTime` be `Instant.now()` (the same object than the one persisted after this method is executed) and `yesterdayScheduledFireTime` be `emailAggregationResources.getLastCronJobRun().getLastRun()`?